### PR TITLE
Only attempt to get lowercase command names.

### PR DESCRIPTION
**Pull request by AndrielChaoti**
This is an overly simple fix for #278

There may be some instances I am missing for this to fully work, and I have not gotten a chance to test this in a live boat just yet. I will update the PR when I've done so.

### CODE REVIEW

**Comment by Kurozume** [comment]
What would happen however in the unlikely event that either we decidd we need a command with an uppercase character or a selfhoster tries to add one?
In its current state this update would mean the bot would never reach aliases with uppercase characters in them

_Sent from my Xperia XZ using [FastHub](https://play.google.com/store/apps/details?id=com.fastaccess.github)_

**Comment by AndrielChaoti** [comment]
True enough. Perhaps having the command registry add command names and aliases in lower case as well?

If it came to the need for commands with uppercase names, then I suppose this would be moot. :)

**Comment by AndrielChaoti** [comment]
I've given this a test on a discord server at this point. I feel that adding the `.toLowerCase` to the command registration system is also necessary before the PR should be merged.

File: FredBoat/src/main/java/fredboat/commandmeta/CommandRegistry.java [hunk 1]
```diff
@@ -81,7 +82,7 @@ public String help(Guild guild) {
 
         CommandEntry(Command command, String name) {
             this.command = command;
-            this.name = name;
+            this.name = name.toLowerCase();
```
Comment (Shredder121) [inline FredBoat/src/main/java/fredboat/commandmeta/CommandRegistry.java]: At this point the `name` had been lowercased already in `registerCommand`

Comment (AndrielChaoti) [reply FredBoat/src/main/java/fredboat/commandmeta/CommandRegistry.java]: so it has... fixed that :smile:

**Comment by Shredder121** [comment]
I'm not really a fan of `toLowerCase` without a `Locale` argument, as that usually fails the ['Turkey test'][turkey-test]
Now this is not really a big problem with server architecture programs, but since the commands are English alphabet only, is it an idea to add a `Locale.ENLGISH`?

Oh okay lol nvm.
[turkey-test]: https://stackoverflow.com/questions/796986/what-is-the-turkey-test

**Pull request merged**